includes/admin/class-sensei-setup-wizard.php

```diff
@@ -417,17 +417,51 @@ public function get_mailing_list_form_data() {
 		];
 	}
 
+	/**
+	 * Get data used for WooCommerce.com purchase redirect for feature installation.
+	 *
+	 * @return array The data.
+	 */
+	public function get_woocommerce_connect_data() {
+
+		$is_woocommerce_installed = Sensei_Utils::is_plugin_present_and_activated( 'Woocommerce', 'woocommerce/woocommerce.php' );
+
+		if ( ! $is_woocommerce_installed ) {
```

Comment:
We might also want to do a version check. Some quick research, I think this functionality was added in WC 3.7.0, but not 100% sure.

Reply:
Added this too above, just for the the `WC_Admin_Addons` usage.
Looks like the auto-install helper itself was integrated around 3.1, not sure what minimum is needed for current WooCommerce.com compatibilty. But the checkout will work without any of that.